### ISSUE TITLE
Failed to Run `docker-compose build dev`

### ISSUE THREAD

**Issue by PIG208** [metacall/faas#2]
When I run `docker-compose build dev`, I got the following errors while building:
```
 > [4/4] RUN metacall npm install       && metacall npm run build:
#8 0.903 npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
#8 3.525 npm WARN tar ENOENT: no such file or directory, open '/metacall/source/node_modules/.staging/type-fest-42e5749e/ts41/snake-case.d.ts'
#8 3.545 npm WARN tar ENOENT: no such file or directory, open '/metacall/source/node_modules/.staging/eslint-7d283110/lib/init/config-file.js'
#8 3.546 npm WARN tar ENOENT: no such file or directory, open '/metacall/source/node_modules/.staging/yargs-bddbd8a5/helpers/helpers.mjs'
...
(250 lines of similar warnings)
...
#8 3.919 npm WARN tar ENOENT: no such file or directory, open '/metacall/source/node_modules/.staging/@typescript-eslint/eslint-plugin-88f7d770/dist/rules/indent-new-do-not-use/BinarySearchTree.js.map'
#8 3.921 npm WARN tar ENOENT: no such file or directory, open '/metacall/source/node_modules/.staging/@typescript-eslint/eslint-plugin-88f7d770/dist/util/collectUnusedVariables.js.map'
#8 4.389 npm ERR! code ENOENT
#8 4.389 npm ERR! syscall spawn git
#8 4.389 npm ERR! path git
#8 4.389 npm ERR! errno ENOENT
#8 4.390 npm ERR! enoent Error while executing:
#8 4.390 npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/metacall/protocol.git
#8 4.390 npm ERR! enoent 
#8 4.391 npm ERR! enoent 
#8 4.391 npm ERR! enoent spawn git ENOENT
#8 4.391 npm ERR! enoent This is related to npm not being able to find a file.
#8 4.391 npm ERR! enoent 
#8 4.474 
#8 4.474 npm ERR! A complete log of this run can be found in:
#8 4.474 npm ERR!     /root/.npm/_logs/2022-03-10T17_54_53_500Z-debug.log
------
failed to solve: rpc error: code = Unknown desc = executor failed running [/bin/sh -c metacall npm install      && metacall npm run build]: exit code: 1
```
I got similar errors for `docker-compose build faas` as well.

Docker version 20.10.12, build e91ed57
Docker Compose version v2.2.3
WSL2 (Ubuntu 20.04.3 LTS)

**Comment by Jasbir96** [comment]
Which version of npm/node you are using?

**Comment by itsritiksingh** [comment]
issue solved in this pr #3
